showcase-docs/dynamic-plugins.md

```diff
@@ -319,12 +336,38 @@ In order to add dynamic plugin support to a third-party plugin, without touching
           disabled: false
   ```
 
-### Example Dynamic plugins
+### Dynamic plugins included in the Showcase container image
+
+The showcase docker image contains a set of ready-to-use dynamic plugins, most of which are disabled by default, since they require some mandatory configuration.
+The list of those plugins can be found in the [`dynamic-plugins.default.yaml`](https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins.default.yaml) file.
+
+At application start, for every plugin disabled by default, the log of the `install-dynamic-plugins` init container of the `backstage` Pod will contain a line like below:
+
+```
+======= Skipping disabled dynamic plugin ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic
+```
+
+Enabling this plugin is very easy: just add a package with the same name and override the `disabled` field in the helm chart values:
+
+```diff
+global:
+  dynamic:
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
++      - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic
++        disabled: false
+```
+
+The plugin default configuration is read from the `dynamic-plugins.default.yaml`, but
+you can still override it by adding a `pluginConfig` entry to the plugin configuration.
```

Comment:
Just a quick question about this, what is the priority for these configurations? Suppose the end user decides to manually configure these values in their `app-config.yaml` via a separate ConfigMap, will the configurations in the `dynamic-plugins.default.yaml` take precedence? 

Also is there a way to enable the plugin BUT not enable the default configurations? Since there may be some configuration conflicts with the default values if those get loaded. 

For example, the `ocm-backend` plugin allows configurations for the entity provider with `catalog.provider.ocm.<arbitrary-key-name>`, however in the default configurations in [dynamic-plugins.default.yaml](https://github.com/janus-idp/backstage-showcase/blob/3a350189e714f5aa4980c1724539f822abcf33f5/dynamic-plugins.default.yaml#L382-L394) this `arbitrary-key-name` is `default`. 

So if this default configuration is loaded and the user tries to override it with their own configurations like below (with a different `arbitrary-key-name` than the one in our default (`default`)), the backend will fail complaining about missing configurations for `ocm.default`. 
```yaml
catalog:
  providers:
     ocm:
        test:
           # other configurations
```

This was the behavior when we had the plugin installed statically and had the "default" config keys defined in the `app-config.yaml` (now [`app-config.deprecated.yaml`](https://github.com/janus-idp/backstage-showcase/blob/3a350189e714f5aa4980c1724539f822abcf33f5/app-config.deprecated.yaml#L233-L238))

Reply:
When I write:

> you can still override it by adding a `pluginConfig` entry to the plugin configuration.

I'm mean:

> replace the whole default configuration fragment by the one you will provide in the `pluginConfig` field.

I hope this answers your last question.

Reply:
Ok then this shouldn't be an issue